backend/src/connectors/v2/authorisation/silly.ts

```diff
@@ -20,23 +27,35 @@ export class SillyAuthorisationConnector implements BaseAuthorisationConnector {
   }
 
   async getEntities(user: UserDoc) {
-    return [toEntity(EntityKind.User, user.dn)]
+    return [toEntity(SillyEntityKind.User, user.dn)]
   }
 
   async getUserInformation(entity: string): Promise<{ email: string }> {
     const entityObject = fromEntity(entity)
-    if (entityObject.kind !== EntityKind.User) {
-      throw new Error('Cannot get user information for a non-user entity')
+    if (entityObject.kind !== SillyEntityKind.User) {
+      throw new Error(`Cannot get user information for a non-user entity: ${entity}`)
     }
     return {
       email: `${entityObject.value}@example.com`,
     }
   }
 
+  async getUserInformationList(entity): Promise<Promise<{ email: string; }>[]> {
+    const entityObject = fromEntity(entity)
```

Comment:
With the suggested changes to `getEntityMembers` below, this function would becomes significantly simpler:

```typescript
{
  const entities = await this.getGroupMembers(entity)
  return entities.map(entity => this.getUserInformation(entity))
}
```